.github/workflows/main.yml

```diff
@@ -8,29 +8,31 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.pair.runner }}
     env:
       MIX_ENV: test
     strategy:
       fail-fast: false
       matrix:
         include:
           - pair:
-              elixir: 1.12.0
-              otp: 22.3
+              elixir: "1.12"
+              otp: "22"
+              runner: "ubuntu-20.04"
```

Comment:
@josevalim Do we still want to support Elixir 1.12, judging that `setup-beam` have minimal support for Erlang OTP 24.2 for its Ubuntu image, see https://github.com/erlef/setup-beam?tab=readme-ov-file#compatibility-between-operating-system-and-erlangotp?

Should we move to Elixir 1.15? If not, Elixir 1.13 is the minimal supported version that works on Erlang OTP 24, see https://hexdocs.pm/elixir/compatibility-and-deprecations.html#between-elixir-and-erlang-otp, which is supported by `setup-beam` `ubuntu-22.04` image.